docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md

```diff
@@ -190,64 +190,113 @@ You can install Cloud Infrastructure Security for AWS from the App Catalog to us
 1. From the **App Catalog**, search for and select **Cloud Infrastructure Security for AWS**.
 1. Click **Install App**.  
 1. In the **Deploy Cloud Infrastructure for AWS** screen, perform the following steps:
-   1. **Select Region**. Select the AWS region where you want to deploy the solution.
+   1. **Select Region**. Select the [AWS region](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html#Concepts.RegionsAndAvailabilityZones.Regions) where you want to deploy the solution. For information about where your Sumo Logic data is stored in AWS, see [Where is My Data Stored?](/docs/get-started/faq/#where-is-my-data-stored)
     :::important
     This step is critical. If you do not select the correct region, you will deploy the solution in the wrong region.
     :::
-   1. **Deploy AWS**. Click the **Deploy AWS Security** button.  <br/><img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-install-1.png')} alt="Deploy Cloud Infrastructure for AWS screen" style={{border: '1px solid gray'}} width="700"/>
+   1. **Deploy AWS**. Click the **Deploy AWS Security** button. <br/><img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-install-1.png')} alt="Deploy Cloud Infrastructure for AWS screen" style={{border: '1px solid gray'}} width="700"/>
```

Comment:
There are two `i. `steps. If you indent the note, might fix it.

<img width="450" alt="Screenshot 2024-03-01 at 9 51 45 AM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/f1a2868e-b2bc-43c2-b8f7-2af9ac3bfff3">

Reply:
Great catch, @kimsauce! And great illustration. 😃